lib/src/layouts/yaru_landscape_layout.dart

```diff
@@ -163,6 +164,15 @@ class _YaruLandscapeLayoutState extends State<YaruLandscapeLayout> {
   }
 
   Widget _buildVerticalSeparator() {
+    // Fix for the ultra white divider on flutter web
+    final darkAndWeb =
+        Theme.of(context).brightness == Brightness.dark && kIsWeb;
```

Comment:
if you swap the conditions around, we avoid an unnecessary inherited theme lookup on non-web platforms
```
kIsWeb && Theme.of(context).brightness == Brightness.dark
```